src/Features/VisualBasic/Portable/Completion/CompletionProviders/PartialTypeCompletionProvider.vb

```diff
@@ -57,29 +63,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
             End If
         End Function
 
-        Private Async Function CreateItemsAsync(document As Document, position As Integer, span As TextSpan, token As SyntaxToken, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of CompletionItem))
+        Private Async Function CreateItemsAsync(document As Document, position As Integer, span As TextSpan, token As SyntaxToken, typeStatement As TypeStatementSyntax, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of CompletionItem))
             Dim semanticModel = Await document.GetSemanticModelForNodeAsync(token.Parent, cancellationToken).ConfigureAwait(False)
 
-            ' Unless the enclosing symbol is already the global namespace, we want to get it's enclosing symbol
-            ' in order to suggest partial types in our namespace.
-            Dim enclosingSymbol = semanticModel.GetEnclosingSymbol(position, cancellationToken)
-            Dim enclosingNamespace = TryCast(enclosingSymbol, INamespaceSymbol)
-            If Not (enclosingNamespace IsNot Nothing AndAlso enclosingNamespace.IsGlobalNamespace) Then
-                enclosingSymbol = enclosingSymbol.ContainingSymbol
-            End If
```

Comment:
These checks were ignored, but they don't seem to produce the correct result anyway. `GetEnclosingSymbol` doesn't produce a useful result. Depending on how far the user has typed, the enclosing symbol may either be the type they are currently declaring, or the containing type, with no obvious way to tell which we got.